tools/matRad_getAlphaBetaCurves.m

```diff
@@ -0,0 +1,94 @@
+function [machine] = matRad_getAlphaBetaCurves(machine,varargin)
+% matRad alpha beta curve calculation tool
+%
+% call
+%   machine.data = matRad_getAlphaBetaCurves(machine.data,pln,cst)
```

Comment:
.data needs to go. Also it shouldn't be "pln" but the model name, right? I also would list all possibilities to call this (1. data only, + cst alone or + modelName alone, and both.

Reply:
I needed the pln for the bioParam.calcLQParameter function, but nothing else, so this is now generated inside of the function.